photon-client/src/main/scala/com/linkedin/photon/ml/Constants.scala

```diff
@@ -12,15 +12,28 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
-package com.linkedin.photon.ml.avro
+package com.linkedin.photon.ml
 
 /**
  * Some commonly used String constants.
  */
 object Constants {
-  protected[avro] val DEFAULT_AVRO_FILE_NAME = "part-00000.avro"
-  protected[avro] val ID_INFO = "id-info"
-  protected[avro] val COEFFICIENTS = "coefficients"
-  protected[avro] val FIXED_EFFECT = "fixed-effect"
-  protected[avro] val RANDOM_EFFECT = "random-effect"
+
+  /**
+   * Delimiter used to concatenate feature name and term into feature key.
+   *
+   * WARNING: This is not visible in println!
+   */
+  val DELIMITER = "\u0001"
+
+  /**
+   * Wildcard character used for specifying the feature constraints. Only the term is allowed to be a wildcard normally
+   * unless one wants to apply bounds to all features in which case both name and term can be specified as wildcards.
+   * Currently, we do not support wildcards in name alone.
+   */
+  val WILDCARD = "*"
+
+  val INTERCEPT_NAME = "(INTERCEPT)"
+  val INTERCEPT_TERM = ""
+  val INTERCEPT_NAME_TERM: String = INTERCEPT_NAME + DELIMITER + INTERCEPT_TERM
```

Comment:
Would be better to use the `getFeatureKey` function here, rather than duplicating key construction. Also, why did it change from `INTERCEPT_KEY` to `INTERCEPT_NAME_TERM`? The name/term thing is a detail of how the key is constructed, but code downstream from this doesn't need to know that.